config/config-revision-gc.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-revision-gc
+  namespace: knative-serving
+data:
+  # Delay after revision creation befor considering it for GC
+  stale-revision-create-delay: 5m
+  # Duration since a route has been pointed at a revision before it should be GC'd
+  stale-revision-timeout: 5m
```

Comment:
I think another useful know would be a minimum number of generations to keep?

e.g. if the current revision is from `Generation: 5` and we've been directed to retain the past `3` generations, then we should only GC the revisions for generations `1-2`.  This could be very useful for rollbacks using "release" mode on Service.

Reply:
Makes sense - added with stale-revision-minimum-generations.